tests/cnf/core/network/internal/define/nad.go

```diff
@@ -90,3 +93,13 @@ func createNadWithMasterPlugin(
 
 	return createdNad, nil
 }
+
+// CreateStaticIPAnnotations creates a static ip annotation used together with the nad in a pod for IP configuration.
+func CreateStaticIPAnnotations(internalNADName, externalNADName string, internalIPAddresses,
```

Comment:
I beleive this func should not be a part of nad.go file. It's related to metallb

Reply:
moved to frrconfig